tox.ini

```diff
@@ -154,18 +154,7 @@ env_dir = {work_dir}/docs
 runner = ignore_env_name_mismatch
 deps =
     -r{tox_root}/requirements.txt
-    # renovate: datasource=pypi
-    Sphinx==6.1.3
-    # renovate: datasource=pypi
-    sphinx_design==0.4.1
-    # renovate: datasource=pypi
-    sphinx-copybutton==0.5.1
-    # renovate: datasource=pypi
-    sphinx-pydantic==0.1.1
-    # renovate: datasource=pypi
-    sphinx-toolbox==0.1.0
-    # renovate: datasource=pypi
-    sphinx-lint==0.6.7
+    -r{tox_root}/docs/.sphinx/requirements.txt
```

Comment:
This is probably the better way to do it overall (or the `docs` requirement we have in many `pyproject.toml` files). However, the lack of `sphinx-lint` in that requirements file is causing the tox linting to fail. However you'd prefer to add it (bring it back in this deps file while keeping the other changes, add it to the sphinx requirements, etc.) I'm happy with.

Reply:
I've just put the requirement straight back in there for now.
When I replace the embedded Sphinx starter pack with a submodule, I will introduce a new requirements file for the documentation.